ml_perf/flags/9/architecture.flags

```diff
@@ -1,6 +1,7 @@
 # Flags that control the model architecture.
 
---conv_width=64
+--conv_width=32
 --fc_width=64
 --trunk_layers=9
 --value_cost_weight=0.25
```

Comment:
Why value_cost_weight is set to 0.25?  I saw it is 1.0 by default, as in Christian's repo

Reply:
This also prevents the value head from overfitting.
Basically, for 9x9 the policy head has 50x-100x more data points than the value head.  For 19x19, the policy head has ~200-250x more data.

When training a supervised network on relatively small numbers of games, DeepMind was "weighting the mean-squared error component by a factor of 0.01", i.e., `value_cost_weight=0.01`

My hunch is that as filter_amount approaches 1, value_cost_weight should approach `1 / (average_game_length)` with some adjustment for window size.